src/status_im/contexts/chat/messenger/messages/delete_message_for_me/events.cljs

```diff
@@ -125,9 +135,14 @@
        (map (fn [message] {:chat-id chat-id :message-id (first message)}))
        (concat acc)))
 
-(rf/defn sync-all
+(defn sync-all
   "Get all deleted-for-me messages that not yet synced with status-go and sync them"
-  {:events [:chat.ui/sync-all-deleted-for-me-messages]}
   [{:keys [db] :as cofx}]
-  (let [pending-sync-messages (reduce-kv filter-pending-sync-messages [] (:messages db))]
-    (apply rf/merge cofx (map #(delete-and-sync % true) pending-sync-messages))))
+  (let [pending-sync-messages (reduce-kv filter-pending-sync-messages [] (:messages db))
+        pending-effects       (map (fn [message]
+                                     (fn [cofx]
+                                       (delete-and-sync cofx [message true])))
+                                   pending-sync-messages)]
+    (apply rf/merge cofx pending-effects)))
```

Comment:
Here we're needing to wrap the call to `delete-and-sync` because `rf/merge` is expecting a sequence of functions that receive a `cofx` object. Normally this behaviour is handled by `rf/defn` but `delete-and-sync` no longer uses that for defining its logic.